checkov/common/bridgecrew/platform_integration.py

```diff
@@ -1041,5 +1042,41 @@ def get_default_headers(self, request_type: str) -> dict[str, Any]:
         logging.info(f"Unsupported request {request_type}")
         return {}
 
+    # Define the function that will get the relay state from the Prisma Cloud Platform.
+    def get_sso_prismacloud_url(self, report_url: str) -> str:
+        if not bc_integration.prisma_api_url:
+            return report_url
+        url_saml_config = f"{bc_integration.prisma_api_url}/saml/config"
+        token = self.get_auth_token()
+        headers = merge_dicts(get_auth_header(token),
+                              get_default_get_headers(self.bc_source, self.bc_source_version))
+
+        request = self.http.request("GET", url_saml_config, headers=headers, timeout=10)
+        if request.status == 401:
+            logging.error(f'Received 401 response from Prisma /login endpoint: {request.data.decode("utf8")}')
+            raise BridgecrewAuthError()
+        elif request.status == 403:
+            logging.error('Received 403 (Forbidden) response from Prisma /login endpoint')
+            raise BridgecrewAuthError()
```

Comment:
@mikeurbanski1 any thoughts about raising an exception here or should we just log it and return the normal URL?

Reply:
I like returning the normal URL. Then at least they can go to it after signing in separately.

Reply:
@mikeurbanski1 @gruebel instead of raising an exception, I could return the normal URL as follow:
```        
request = self.http.request("GET", url_saml_config, headers=headers, timeout=10)  # type:ignore[no-untyped-call]
if request.status >= 300:
  return report_url
```

With the problem that we don't trigger an exception if the BC_API_KEY is not provided. I am not sure this is better. What do you think ?

Reply:
@mikeurbanski1, of course, you are correct. I committed the change.